CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 
 ## Breaking changes
 
+- Removed multiple docker images and reduced it to a single docker image tagged `pybamm/pybamm:latest` with all the solvers (`IDAKLU` and `JAX`) contained in it by default ([#3992](https://github.com/pybamm-team/PyBaMM/pull/3961))
```

Comment:
Just rephrasing this sentence a bit to make it clearer + the PR number was incorrect

```suggestion
- Removed multiple Docker images. Here on, a single Docker image tagged `pybamm/pybamm:latest` will be provided with both solvers (`IDAKLU` and `JAX`) pre-installed. ([#3992](https://github.com/pybamm-team/PyBaMM/pull/3992))
```